static/docs/user-guide/experiments/branches.md

```diff
@@ -0,0 +1,119 @@
+# How to Manage Experiments by Branches
+
+You can use a different Git branch for each experiment.
+
+<p align="center">
+<img src="/static/img/user-guide/experiments/branches.png" />
+</p>
+
+This is usually more flexible than managing experiments by tags, since you can
+easily base a new experiment on any of the previous experiments.
+
+## Examples
+
+An example of managing experiments by branches can be seen on the
+[Deep Dive Tutorial](https://dvc.org/doc/tutorials/deep/reproducibility).
+
+These interactive tutorials also manage experiments by branches:
+
+- [Pipelines](https://katacoda.com/dvc/courses/tutorials/pipelines) - Using DVC
+  commands to build a simple ML pipeline.
+- [MNIST](https://katacoda.com/dvc/courses/tutorials/mnist) - Classify images of
+  hand-written digits using the MNIST dataset.
+
+## How it works
+
+### Commit and branch
+
+Let's say that we are working on the branch `master` and at the end of the
+experiment we want to save it on a branch named `unigrams`. We can do it like
+this:
+
+```dvc
+$ git commit -am 'Evaluate'
+$ dvc commit   # just to make sure all the data is committed
+$ git checkout -b unigrams
+$ git checkout master
+```
+
+Now we can continue working on `master` for another experiment. When we are done
+we can create another branch for it same as above.
+
+### New experiment based on another one
+
+Suppose that we want to start a new experiment based on another one, instead of
+starting from `master`. We can switch first to that branch and then start a new
+experiment on top of it:
+
+```dvc
+$ git checkout unigrams
+$ dvc checkout
```

Comment:
why not `git checkout --branch bigrams unigrams` (go to experiment of brigams based on the unigrams)

Reply:
This would depend on your previous suggestion, https://github.com/iterative/dvc.org/pull/828#discussion_r359003964 ? Perhaps you can replace these 2 comments for a single one for the full thing? 🙂